docs/content/docs/guide/authoringprs.md

```diff
@@ -149,6 +149,7 @@ The fields available are :
 * `target_branch`: The branch we are targeting.
 * `source_branch`: The branch where this pull_request come from. (on `push` this
   is the same as `target_branch`).
+* `repo_url`: The full url of the repository.
```

Comment:
i think it may be better to call it target_url just to be consistent and maybe add source_url in case of pull request so we know the fork come from, should be the same as target_url on push

Reply:
done the changes